### DISCUSSION TITLE
Upload

### DISCUSSION THREAD

**Discussion by boxadesign** [ONSdigital/design-system#97] [Feedback]
Use this place to discuss the [upload component](https://ons-design-system.netlify.com/components/upload) in the [ONS Design System](https://ons-design-system.netlify.com)

**Comment by jrbarnes9** [comment]
Empty duplicate field markup causing unwanted margin in Upload example

<img width="1215" alt="Screenshot 2019-11-29 at 15 22 15" src="https://user-images.githubusercontent.com/43346934/69878197-2315bc00-12bc-11ea-8ea9-a405e084e750.png">

**Reply by jrbarnes9** [reply]
#667 resolved

**Comment by jrbarnes9** [comment]
Upload component doesn't call `onsField` or `onsFieldset` so in turn, doesn't call `onsError`, so the validation pattern can't be used with it.

**Reply by jrbarnes9** [reply]
#1569 raised
